applications/MeshingApplication/tests/test_remesh_sphere.py

```diff
@@ -269,6 +270,7 @@ def test_remesh_sphere_skin(self):
         }
         """)
 
+        check_parameters["input_file_name"].SetString(file_path + "/" + check_parameters["input_file_name"].GetString())
```

Comment:
please use os.path.join, such things fail in windows

Reply:
You can suggest a change with the new GitHub interface, we can try

Reply:
```suggestion
        check_parameters["input_file_name"].SetString(file_path + "/" + check_parameters["input_file_name"].GetString())
```

Reply:
I saw that but was too lazy to try
Some other time :)

Reply:
```suggestion
        check_parameters["input_file_name"].SetString(os.path.join(file_path, check_parameters["input_file_name"].GetString()))
```